services/consensusalgo/leanhelixconsensus/service.go

```diff
@@ -0,0 +1,143 @@
+package leanhelixconsensus
+
+import (
+	"context"
+	"github.com/orbs-network/lean-helix-go"
+	lhprimitives "github.com/orbs-network/lean-helix-go/primitives"
+	"github.com/orbs-network/orbs-network-go/instrumentation/log"
+	"github.com/orbs-network/orbs-network-go/instrumentation/metric"
+	"github.com/orbs-network/orbs-spec/types/go/primitives"
+	"github.com/orbs-network/orbs-spec/types/go/protocol"
+	"github.com/orbs-network/orbs-spec/types/go/protocol/consensus"
+	"github.com/orbs-network/orbs-spec/types/go/services"
+	"github.com/orbs-network/orbs-spec/types/go/services/gossiptopics"
+	"sync"
+	"time"
+)
+
+var LogTag = log.Service("consensus-algo-lean-helix")
+
+type service struct {
```

Comment:
it's not really a `service`, is it? more of an `adapter`

Reply:
or `bridge`

Reply:
Consensus algo is a service like any other. There was no need for a separate adapter so I merged that into the service.